src/containers/Reports/Reports.js

```diff
@@ -52,41 +54,55 @@ class Reports extends Component {
         const formatted = Object.keys(data['callersByMonth']).map((el)=>{
             const numCallers = data['callersByMonth'][el]
             const numCalls = data['callsByMonth'][el] || 0
-            return {date: el, Callers: numCallers, Calls: numCalls};
-        });
+            const completed = (numCalls/numCallers * 100).toFixed(1) + "%"
+            return {date: el, Callers: numCallers, Calls: numCalls, Completed: completed};
+        });  
         return formatted.slice(0, 11);
     }
 
+    
     render() {
         const statistics = this.state.statistics;
         const antIconBig = <Icon type="loading" style={{ fontSize: 28 }} spin />
         const antIconHuge = <Icon type="loading" style={{ fontSize: 72 }} spin />
         const districtTitle = this.props.district ? `${displayName(this.props.district)} ` : "";
 
+        let completionRate = 0; 
+        if(statistics && statistics.totalCalls > 0 && statistics.totalCallers > 0)
```

Comment:
A note on curly braces and code formatting:

https://avandeursen.com/2014/02/22/gotofail-security/

tldr: use what the language provides you for maximum clarity in your code

It's a good idea to use a code linter to automatically do this for you. I recommend adding Prettier to your text editor for this project. https://prettier.io/docs/en/editors.html